pkg/collector/corechecks/system/cpu_windows.go

```diff
@@ -83,6 +83,7 @@ func (c *CPUCheck) Run() error {
 
 	nbCycle := t.Total() / c.nbCPU
 
+	sender.Gauge("system.cpu.num_cores", c.nbCPU, "", nil)
```

Comment:
nit: move this after the `if c.lastNbCycle` to have the same code structure on both Windows and Linux